Scripts/Services/Assistant/V1/Model/DialogNode.cs

```diff
@@ -1,5 +1,5 @@
 /**
-* (C) Copyright IBM Corp. 2018, 2020.
+* (C) Copyright IBM Corp. 2019, 2020.
```

Comment:
strange that these are being changed

Reply:
I am not sure how this is happening. Looking into it

Reply:
strangely the command in lint-copyrights gives 2019
```
git log --pretty=format:"%ad" --follow -M95% --date=format:%Y Scripts/Services/Assistant/V1/AssistantService.cs
```

Reply:
In this case, it is correct: https://github.com/watson-developer-cloud/unity-sdk/commits/52f0cf940865fc28cbe14833c69bed8293149a1a/Scripts/Services/Assistant/V1/Model/DialogNode.cs

Theses got refactored in 2019: https://github.com/watson-developer-cloud/unity-sdk/tree/v2.13.2 in 2.14.0